v1/api/swagger/components/schemas/NewHost.yaml

```diff
@@ -149,3 +149,11 @@ properties:
     example:
       serviceType: BMaaS
       region: us-west-2
+  WWPNs:
+    type: array
+    description: FC HBA world wide port names
+    items:
+      type: string
+    example:
+      - 20:31:01:02:ad:f2:a8:e5
+      - 21:31:01:02:ad:f2:a8:e5
```

Comment:
@chitranm This should be removed from here since this is for creating new Hosts. WWPNs are h/w info that the user should not provide. Sorry I missed this previously.

Reply:
Thanks @mchuang3. So I should remove it only from NewHost.yaml right? And keep the changes in Host.yaml as is?